packages/atomic/src/components/facets/atomic-category-facet/atomic-category-facet.tsx

```diff
@@ -204,7 +216,7 @@ export class AtomicCategoryFacet
         hasActiveValues={this.facetState.hasActiveValues}
         deselectAll={() => this.facet.deselectAll()}
       >
-        {this.facetSearch.render()}
+        {this.enableFacetSearch ? this.facetSearch.render() : null}
```

Comment:
would this work?

```tsx
{this.facetSearch?.render()}
```

also here at the line
```
private facetSearch!: FacetSearch;
```
should be 
```
private facetSearch?: FacetSearch;
```
since it can be undefined